cluster/search/client.py

```diff
@@ -39,7 +43,26 @@ def upload(self, documents):
             doc["_index"] = self.index
         return bulk(self.es, documents)
 
-    def search(self, query, limit=100, only_documents=True):
+    def update(self, query, scroll, updater):
```

Comment:
Nice, I thought we had said elasticsearch didn't have support for this in the past?